src/main/java/com/exasol/dbbuilder/dialects/exasol/AdapterScript.java

```diff
@@ -53,7 +48,138 @@ public String getContent() {
+        protected Schema parentSchema;
+        protected String name;
+        protected boolean owned;
+        private ExasolImmediateDatabaseObjectWriter writer;
+        private Language language;
+        private String content;
+        private String debuggerConnection;
+
+        private Builder() {
+        }
+
+        /**
+         * Set the parent Schema.
+         * 
+         * @param parentSchema parent schema
+         * @return self
+         */
+        public Builder parentSchema(final Schema parentSchema) {
+            this.parentSchema = parentSchema;
+            return this;
+        }
+
+        /**
+         * Set the {@link ExasolImmediateDatabaseObjectWriter}.
+         * 
+         * @param writer {@link ExasolImmediateDatabaseObjectWriter}
+         * @return self
+         */
+        public Builder writer(final ExasolImmediateDatabaseObjectWriter writer) {
+            this.writer = writer;
+            return this;
+        }
+
+        /**
+         * Set the language of the adapter script.
+         * 
+         * @param language language of the adapter script
+         * @return self
+         */
+        public Builder language(final Language language) {
+            this.language = language;
+            return this;
+        }
+
+        /**
+         * Set the content of the adapter script.
+         * 
+         * @param content script content
+         * @return self
+         */
+        public Builder content(final String content) {
+            this.content = content;
+            return this;
+        }
+
+        /**
+         * Set an optional connection to a debugger. See {@link ExasolConfiguration#isAdapterScriptDebuggingEnabled()}
+         * 
+         * @param debuggerConnection optional connection to a debugger
+         * @return self
+         */
+        public Builder debuggerConnection(final String debuggerConnection) {
+            this.debuggerConnection = debuggerConnection;
+            return this;
+        }
+
+        /**
+         * Set the name of the adapter script..
+         * 
+         * @param name name of the adapter script
+         * @return self
+         */
+        public Builder name(final String name) {
+            this.name = name;
+            return this;
+        }
+
+        /**
+         * Build the {@link AdapterScript}.
+         * 
+         * @return built {@link AdapterScript}.
+         */
+        public AdapterScript build() {
+            requireNotNull(this.writer, "writer");
+            requireNotNull(this.parentSchema, "parentSchema");
+            requireNotNull(this.name, "name");
+            requireNotNull(this.language, "language");
+            requireNotNull(this.content, "content");
+            return new AdapterScript(this.writer, this.parentSchema, this.name, this.language, this.content,
+                    this.debuggerConnection);
+        }
+
+        private void requireNotNull(final Object object, final String name) {
```

Comment:
There is a method in the standard Library for this purpose: https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/util/Objects.html#requireNonNull(T,java.lang.String)

But your variant has better output. So :+1: